docs/ref/csrf.txt

```diff
@@ -99,14 +99,14 @@ You can acquire the token like this:
         }
         return cookieValue;
```

Comment:
```suggestion
```

Reply:
Maybe an explicit `return null;` instead? To follow PEP8 (section "Be consistent in return statements"). Event though this is JavaScript? :D

Reply:
Certainly not. 'null' is a very strange object indeed